### PR TITLE
Add RBAC manifests for ExternalDNS

### PR DESCRIPTION
**Pull request by linki**
Add RBAC manifests for ExternalDNS.

### CODE REVIEW

**Comment by linki** [comment]
Note, this also wraps the RBAC rules in the `ConfigItem` check to make it consistent but different than the other places so far. Should be fine though.

File: cluster/manifests/external-dns/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  namespace: kube-system
+  labels:
+    application: external-dns
+---
+# allows to list services and ingresses
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: external-dns
+  namespace: kube-system
```
Comment (mikkeloscar) [inline cluster/manifests/external-dns/rbac.yaml]: `ClusterRoles` or `ClusterRoleBindings` are not namespaced. I don't know if this makes it break or it just ignores it.

Comment (linki) [reply cluster/manifests/external-dns/rbac.yaml]: That doesn't make sense 😆 Fixed.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**